kb/skills/init/SKILL.md

```
@@ -0,0 +1,102 @@
+
+If a kb already exists at the target (has `AGENTS.md` or `memory/`), this skill
+is still safe to run — see Idempotency below — but check first whether the user
+actually wants `dream` (consolidate existing notes) instead of `init`.
+
+## Steps
+
+1. **Resolve the target path.** Default `$HOME/kb`. Use `$KB_DIR` if set, or a
+   path the user gave explicitly. Confirm the path with the user if ambiguous.
+2. **Scaffold.** Run the bundled script with an absolute path (it does not
+   expand `~`):
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)